### ISSUE TITLE
Option to start in manual mode.

### ISSUE THREAD

**Issue by raminnoodle** [WebsiteBeaver/capacitor-document-scanner#1]
Would it be possible to add an option to start the scanner in manual mode.

**Comment by dmarcs** [comment]
@raminnoodle I agree that being able to start the scanner in manual mode is a good idea. Unfortunately VisionKit doesn't have that option. Keep in mind that manual is the only option on Android.

**Comment by dmarcs** [comment]
@RickGroenewegen that is possible, but not how I implemented it because the native iOS document scanner is better.

**Comment by obvar** [comment]
Is there an update on this? Is there a way to force manual mode on iOS? I may have to find a different plugin as this is a deal-breaker.
